### PR TITLE
doc: remove you/your from current docs

### PR DESCRIPTION
**Pull request by heyitsanthony**

### CODE REVIEW

**Comment by gyuho** [comment]
LGTM. Thanks.

File: Documentation/dev-internal/discovery_protocol.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ UUID=$(uuidgen)
 
 ### Specifying the Expected Cluster Size
 
-You need to specify the expected cluster size for this discovery token. The size is used by the discovery service to know when it has found all members that will initially form the cluster.
+THe discovery token expects a cluster size that must be specified. The size is used by the discovery service to know when it has found all members that will initially form the cluster.
```
Comment (xiang90) [inline Documentation/dev-internal/discovery_protocol.md]: The instead of The

File: Documentation/op-guide/runtime-reconf-design.md [hunk 2]
```diff
@@ -4,47 +4,47 @@ Runtime reconfiguration is one of the hardest and most error prone features in a
 
 Read on to learn about the design of etcd's runtime reconfiguration commands and how we tackled these problems.
 
-## Two Phase Config Changes Keep you Safe
+## Two phase config changes keeps the cluster
```
Comment (xiang90) [inline Documentation/op-guide/runtime-reconf-design.md]: Two phase config changes keep the cluster safe

**Comment by xiang90** [comment]
LGTM!!

**Pull request merged**